plugins/tagvalidator/resoto_plugin_tagvalidator/config.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass, field
+from typing import ClassVar, Dict, Union, List
+from resotolib.utils import parse_delta
+
+
+default_config = {
```

Comment:
Looks like this could be a typed config?

Reply:
Yes, it could, it should.